.github/workflows/ci-clang-format.yml

```diff
@@ -78,6 +78,8 @@ jobs:
         DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
         CC: clang-9
         CXX: clang++-9
+        CI_TRIGGER: ${{ github.event_name }}
```

Comment:
We don't need to pass `CI_TRIGGER` and `CI_BRANCH` in all CI workflows, but it seems a good idea to be consistent and do it anyway.

Reply:
Sure, though it would be nice to have some way to share it...I don't think there is one that's worth the trouble though.